targets/REV3-INA138/INA138_main.cpp

```diff
@@ -0,0 +1,38 @@
+/**
+* This is a basic sample to show how the current sensing feature of the
+ * LVSS using the INA138 IC
+*/
+
+#include <EVT/io/UART.hpp>
+#include <EVT/io/pin.hpp>
+#include <EVT/manager.hpp>
+#include <EVT/utils/log.hpp>
+#include <EVT/utils/time.hpp>
+#include <LVSS.hpp>
+
+namespace IO = EVT::core::IO;
+namespace DEV = EVT::core::DEV;
+namespace time = EVT::core::time;
+namespace log = EVT::core::log;
+namespace types = EVT::core::types;
+using namespace std;
```

Comment:
`using namespace` can cause a lot of issues, so you generally don't want to do that
```suggestion
```